tests/cisco_ios/show_cdp_neighbors/cisco_ios_show_cdp_neighbors_3.raw

```diff
@@ -0,0 +1,28 @@
+Capability Codes: R - Router, T - Trans Bridge, B - Source Route Bridge
+S - Switch, H - Host, I - IGMP, r - Repeater, P - Phone,
+D - Remote, C - CVTA, M - Two-port Mac Relay
+
+Device ID Local Intrfce Holdtme Capability Platform Port ID
+SEP000000000000 Gig 1/0/16 155 H P IP Phone port 1
+AP000000000 Gig 1/0/45 158 T B I AIR-CAP35 Gig 0
+AP000000001 Gig 1/0/46 173 T B I AIR-CAP35 Gig 0
+WAAS0000000 Gig 1/0/1 138 T OE474 InlinePort 1/1/lan
+WAAS0000000 Gig 1/0/40 138 H OE474 Gig 1/0
+router.domain.local

```

Comment:
I assume these should be on the same line?

Reply:
I was looking at something else and noticed this may be the way the output is provided back. We would need to account for this within the template to make sure the data is what we need.

I will find the example and post it back, but I think it may have been a similar template to this, but different OS.